examples/ota-provider-app/esp32/main/BdxOtaSender.cpp

```diff
@@ -133,7 +133,7 @@ void BdxOtaSender::HandleTransferSessionOutput(TransferSession::OutputEvent & ev
             mTransfer.AbortTransfer(StatusCode::kUnknown);
             return;
         }
-
+        ChipLogError(BDX, "kQueryReceived PacketBufferHandle allocated address %p", &blockBuf);
```

Comment:
This logs the address of the handle object on the stack, not the address of the packet buffer itself.